RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -598,54 +598,79 @@ void auto_scepterSkills()
-		if(canUse($skill[Aug. 30th: Beach Day!]) && !get_property("_aug30Cast").to_boolean())
+		use_skill($skill[Aug. 24th: Waffle Day!]); //get some waffles to hopefully change some bad monsters to better ones
+	}
+	if(canUse($skill[Aug. 28th: Race Your Mouse Day!]) && !get_property("_aug28Cast").to_boolean() && pathHasFamiliar())
+	{
+		familiar hundred_fam = to_familiar(get_property("auto_100familiar"));
+		if (((in_avantGuard() && in_hardcore()) || (hundred_fam != $familiar[none] && (isAttackFamiliar(hundred_fam) || hundred_fam.block))) && have_familiar(findRockFamiliarInTerrarium()))
 		{
-			use_skill($skill[Aug. 30th: Beach Day!]); //Rollover adventures
+			use_familiar(findRockFamiliarInTerrarium());
+			use_skill($skill[Aug. 28th: Race Your Mouse Day!]); //Fam equipment to lower weight of attack familiar or Burly bodyguard (Avant Guard) for Gremlins
 		}
-		if(canUse($skill[Aug. 28th: Race Your Mouse Day!]) && !get_property("_aug28Cast").to_boolean() && pathHasFamiliar())
+		else if((!auto_hasStillSuit() && item_amount($item[Astral pet sweater]) == 0) || in_small())
 		{
-			familiar hundred_fam = to_familiar(get_property("auto_100familiar"));
-			if (((in_avantGuard() && in_hardcore()) || (hundred_fam != $familiar[none] && (isAttackFamiliar(hundred_fam) || hundred_fam.block))) && have_familiar(findRockFamiliarInTerrarium()))
+			if(!is100FamRun())
 			{
-				use_familiar(findRockFamiliarInTerrarium());
-				use_skill($skill[Aug. 28th: Race Your Mouse Day!]); //Fam equipment to lower weight of attack familiar or Burly bodyguard (Avant Guard) for Gremlins
+				use_familiar(findNonRockFamiliarInTerrarium()); //equip non-rock fam to ensure we get tiny gold medal
 			}
-			else if((!auto_hasStillSuit() && item_amount($item[Astral pet sweater]) == 0) || in_small())
+			else
 			{
-				if(!is100FamRun())
-				{
-					use_familiar(findNonRockFamiliarInTerrarium()); //equip non-rock fam to ensure we get tiny gold medal
-				}
-				else
-				{
-					use_familiar(hundred_fam); // assuming non-rock familiar
-				}
-				use_skill($skill[Aug. 28th: Race Your Mouse Day!]); //Fam equipment
+				use_familiar(hundred_fam); // assuming non-rock familiar
 			}
+			use_skill($skill[Aug. 28th: Race Your Mouse Day!]); //Fam equipment
 		}
 	}
-	//Day 2+ skills
-	if(my_daycount() >= 2)
+	//see how much mana cost reduction we can get (up to 3mp)
+	maximize("-mana cost", true);
+	int manaCostMaximize = numeric_modifier("Generated:_spec", "Mana Cost");
+	if(manaCostMaximize < 3 && canUse($skill[Aug. 30th: Beach Day!]) && !get_property("_aug30Cast").to_boolean() && get_property("_augSkillsCast").to_int()< 5)
+	{
+		use_skill($skill[Aug. 30th: Beach Day!]); //For -MP (and Rollover Adventures)
+	}
+}
+
+void auto_scepterRollover()
+{
+	//We don't want the baywatch if our accessory slots are already filled with > 7 adventure items or we if one of the slots is the counterclockwise watch
+	boolean noWatch = ((numeric_modifier(equipped_item($slot[acc1]),"Adventures") > 7 &&
```

Comment:
all these should be >= 7. If all our accessories are 7, or if we have a watch giving 7 somehow, we shouldn't get a baywatch because it's no better than what we have.